ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -356,6 +356,7 @@ Feature: Notification creation
     And I enter 2019 into 'FormattedEventDate_Year'
     And I select Treatment outcome for 'treatmentevent-type'
     And I select Cured for 'treatmentoutcome-type'
+    And I wait

```

Comment:
I really dislike these waits, because they are both slow and flaky - I'm hoping that we can get rid of them all at some point.

Obviously we do need to do something about the flakiness that is still present in our UI tests, but I don't think this is the place to do it, so I think we should revert this for now, and open a separate ticket.

Reply:
I'll raise a ticket for it, changes reverted